FinniversKit/Sources/Components/BadgeView/BadgeView.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import UIKit
+
+public class BadgeView: UIView {
```

Comment:
Can this be `final class` since it's not open for subclassing outside the framework?

Reply:
Sure! Done ✅